websub-ballerina/commons.bal

```diff
@@ -115,6 +130,17 @@ public type SubscriptionChangeResponse record {|
     http:Response response;
 |};
 
-isolated function isSuccessStatusCode(int statusCode) returns boolean {
-    return (200 <= statusCode && statusCode < 300);
-}
+# Record representing the query-parameters retrieved from the `http:Request`
+# 
+# + hubMode         - value for the `hub.mode` parameter
+# + hubTopic        - value for the `hub.topic` parameter
+# + hubChallenge    - value for the `hub.challenge` parameter
+# + hubLeaseSeconds - value for the `hub.lease_seconds` parameter
+# + hubReason       - value for the `hub.reason` parameter
```

Comment:
Is this how the vs-code formats ?

I think others follow the below convention.

`# + hubMode - value for the `hub.mode` parameter`